### PR TITLE
Improve appearance of playlist items display

### PR DESCRIPTION
**Pull request by peppy**
Before:
20210202 165826 (dotnet)

20210202 170218 (dotnet)

After:

20210202 170910 (dotnet)

20210202 170756 (dotnet)

The readability of text is not perfect but about as far as I'm willing to push it (at least the text doesn't look weirdly pixelated any more).

### CODE REVIEW

File: osu.Game/Graphics/UserInterface/GrayButton.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class GrayButton : OsuAnimatedButton
+    {
+        protected SpriteIcon Icon;
+        protected Box Background;
```
Comment (smoogipoo) [inline osu.Game/Graphics/UserInterface/GrayButton.cs:15]:
```suggestion
        protected SpriteIcon Icon { get; private set; }
        protected Box Background { get; private set; }
```

**Review by smoogipoo — DISMISSED** [review]
looks good otherwise

**Review by bdach — DISMISSED** [review]

**Comment by bdach** [comment]
Looks like an actual headless-only test failure...

**Comment by bdach** [comment]
Never mind, it wasn't headless-only. The additional button added in `DrawableRoomPlaylistWithResults.CreateButtons()` wasn't specifying the correct anchor. I've fixed this in a hopefully more robust way.

**Review by bdach — APPROVED** [review]

**Pull request merged**